lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+class Menu 
+  attr_reader :menu_items
+  def initialize
+    @menu_items = {
+      "tori" => 6,
+      "squid" => 5,
+      "pork" => 8,
+      "ginger" => 11,
+      "king" => 12,
+      "chicken" => 10
+    }
+  end
+
+  def display
+    @menu_items.map { |dish, price| "#{dish}: £#{price}" }.join(", ")
```

Comment:
A clever way of displaying the menu! Using the map method means that it will create a new array and not affect the original, while also not having the be stored anywhere because Ruby returns the last line of code in a method.